### ISSUE TITLE
create a video for testing for demo mode

### ISSUE THREAD

**Issue by meleu** [RetroAchievements/docs-archive#12]
maybe adapting this forum post for a doc page:
http://retroachievements.org/viewtopic.php?t=81

# update

It would be cool if we had a video showing techniques to find the RAM address for demo mode. In some games it's really easy to find, but in other's it can be tricky. The video should show examples on various games, the simple and the complex ones.

**Comment by kdecks** [comment]
Currently best covered: https://github.com/RetroAchievements/docs/wiki/Tips-and-Tricks#memory-digging-tips

**Comment by meleu** [comment]
I agree that it wouldn't be simple to explain "how to detect demo mode" via text. A video would work better, but I'm afraid my foreigner accent is not that good...

I'll change the title to make it clear that we want a video and keep this issue as open.

**Comment by kdecks** [comment]
A video? You're opening a Pandora's box for making video solutions for all our docs.. Wouldn't hurt.

**Comment by meleu** [comment]
the "video tutorials box" was never closed. Every volunteer is very welcome to create them sharing their techniques of achievement creation.

by the way, Magus created a video to illustrate his technique of [Creating a Timer with ResetIf Hits based on the Speed of the Game](https://github.com/RetroAchievements/docs/wiki/Creating-a-Timer-with-ResetIf-Hits-based-on-the-Speed-of-the-Game):
https://youtu.be/6PpdG04tM4s
